app/src/main/java/com/example/mindyourpet/AddReminderFragment.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.mindyourpet
+
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import android.widget.Spinner
+import android.widget.TimePicker
+import androidx.annotation.RequiresApi
+import androidx.fragment.app.Fragment
+import com.example.mindyourpet.database.Reminder
+import com.example.mindyourpet.database.ReminderDatabaseDao
+import kotlinx.coroutines.runBlocking
+
+class AddReminderFragment : Fragment() {
+
+    private lateinit var reminderDao: ReminderDatabaseDao
+
+    @RequiresApi(Build.VERSION_CODES.M)
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+
+        //find view by id in fragment
+        val view: View = inflater.inflate(R.layout.add_reminder, container, false)
+        val saveBtn = view.findViewById<Button>(R.id.save_button)
```

Comment:
The program crashed when I clicked on `saveBtn`.

Reply:
FYI, this still crashes. I also noticed that the UI does not allow specifying day of week/month.

Reply:
I had told Ren that she could leave the implementation unfinished due to her needing to attend to other projects.